src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -384,6 +386,13 @@ public boolean shouldEmailRespondentSolicitor(Map<String, Object> caseData) {
             && !NO_VALUE.equalsIgnoreCase(nullToEmpty(caseData.get(RESP_SOLICITOR_NOTIFICATIONS_EMAIL_CONSENT)));
     }
 
+    public boolean shouldEmailContestedAppSolicitor(Map<String, Object> caseData) {
```

Comment:
change method name please to something like isApplicantSolicitorEmailComunicationEnabled